doc/1/plugins/plugin-context/customize-errors/index.md

```diff
@@ -0,0 +1,61 @@
+---
+code: true
+type: page
+title: customize errors
+---
+
+# Customize errors
+
+When creating your own Kuzzle plugin, you can use the `errorsManager` to have customizable errors.
+
+In order to define it, you have to write them inside the [manifest.json](https://docs.kuzzle.io/core/1/plugins/guides/manual-setup/prerequisites/#manifest-json) in a `errors` field.
+
+Your manifest would be something like :
+```
+{
+    "name": "kuzzle-plugin-xxx",
+    "kuzzleVersion": ">=1.0.0 <2.0.0",
+    "privileged": false,
+    "errors": {
+        "some_error": {
+            "code": 1,
+            "message": "Some error occurred: %s",
+            "class": "BadRequestError"
+	},
+        "some_other_error": {
+            "code": 2,
+            "message": "Some other error occurred: %s",
+            "class": "ForbiddenError"
+	}
+    }
+}
+```
+
+It is exposed as `errorsManager` in the [PluginContext](https://docs.kuzzle.io/core/1/plugins/plugin-context/accessors/intro/).
+The `errorsManager` provides two functions:
+- To throw : `context.errorsManager.throw(error, placeholders);`.
+- To get the built error: `context.errorsManager.getError(error, placeholders);`
+
+Also, you can give more precision by giving a [`subdomain`](https://docs.kuzzle.io/core/1/plugins/plugin-context/errors/kuzzleerror/) to your plugin in you [kuzzlerc file](https://docs.kuzzle.io/core/1/plugins/guides/manual-setup/config/). It would be like :
```

Comment:
IMHO the subdomain should also be precised in the manifest

Reply:
We discussed here
https://kaliop.slack.com/archives/G4LN18NLQ/p1566564559390200
And we conclude that it should be in the kuzzlerc